packages/primitives/radio/src/Root.tsx

```diff
@@ -13,15 +13,21 @@ const styles = normalizeWebStyle({
   },
 })
 
+const invisibleStyles = normalizeWebStyle({
+  display: 'none',
```

Comment:
This would change the size of the component right?

Reply:
this was changed now. But yes. It changes the size of the component...
`isVisible`, at this point, is intended for debugging purposes